app/services/CommonTaskListService.scala

```diff
@@ -16,35 +16,32 @@
 
 package services
 
+import cats.data.EitherT
 import config.AppConfig
+import connectors.errors.ApiError
 import models.get.{AllCISDeductions, CISSource}
+import models.mongo.JourneyAnswers
```

Comment:
This is because these files are covered by the backend endpoints ticket which is blocking this one. I have not pushed them because I will need to rebase them into my branch before I can make this PR ready for review